### PR TITLE
add axios, basic setup for articles component

### PR DESCRIPTION
**Pull request by adampaulsackfield**

### CODE REVIEW

File: src/components/Articles.jsx [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+	}, []);
+
 	return (
-		<div>
+		<div className='h-screen'>
 			<h1>Articles</h1>
+
+			{articles &&
+				articles.map((article) => {
+					return <p>Article: {article.article_id}</p>;
```
Comment (jadelandeg) [inline src/components/Articles.jsx:24]: would show more than just article ID here

**Review by jadelandeg — CHANGES_REQUESTED** [review]
Links are looking good and basic set up for articles is working. Hamburger menu doesn't seem to do anything for me? Just changes from cross to lines when I click it but nothing more. Not sure if you want more functionality yet or just got it there ready?

File: src/components/Articles.jsx (hunk 1 above)
Comment (adampaulsackfield) [reply src/components/Articles.jsx:24]: I do plan on showing more. I just wanted to ensure was on correct path before moving forward. As for the hamburger menu, I am not sure as locally it shows the menu when you click it. I am confused as to why it doesn;t for you.

**Review by jadelandeg — APPROVED** [review]
approving based on video adam sent, looks like tailwind isn't working in sandbox or something

**Pull request merged**